.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
+
+For questions of the type: "How can this be done?", please consider posting them on Stackoverflow.
+
+For everything else ignore this template.
+
+## Bug report
+
+- [ ] Start a new R session
```

Comment:
Stuff in the .RData should not compromise mlrs functionality. Loaded namespaces are another thing.